### PR TITLE
M058: OOI extern services integration to coi-services eoi

### PR DESCRIPTION
**Pull request by birdage**
M058:
Integration of externalization services to coi-services, requires update to name in pyon.yml for tests. Allows EOI services to piggy back on CRUD operations in dataset management service. To actually leverage the services it requires that the importer_service be setup and running correctly, as well as other eoi services and that the eoi.meta.use_eoi_services be set to true in the pyon.yml file.

### CODE REVIEW

**Comment by birdage** [comment]
@jamie-cyber1 @oceanzus this is interesting as the fail basically means it could not get the properties from the pyon.yml file or from pyon.local.yml.

**Comment by mmeisinger** [comment]
It is not allowed to use CFG in static module code. At the time when it is executed, it is not yet loaded correctly. Please place the access of CFG into the setUp() of the test

**Comment by birdage** [comment]
@mmeisinger roger

**Comment by birdage** [comment]
currently looking in to the fail.

**Comment by birdage** [comment]
@jamie-cyber1 tests pass now, ready for merge.

File: ion/services/eoi/test/test_table_loader.py [hunk 1]
```diff
@@ -0,0 +1,399 @@
...
+        print "--------------------------------"
+
+        #verify table exists in the DB (similar to above)
+        # ....code...
+
+        # check that the geoserver layer exists as above
+        # ... code ....
+
+        # make a WMS/WFS request...somet like this (or both)
+        url = 'http://localhost:8080/geoserver/geonode/ows?service=WFS&version=1.0.0&request=GetFeature&typeName=geonode:ooi_' + dataset_id + '_ooi&maxFeatures=1&outputFormat=csv'
```
Comment (jamie-cyber1) [inline ion/services/eoi/test/test_table_loader.py]: Why is it point to localhost? Shouldn't it read from CFG?

Comment (birdage) [inline ion/services/eoi/test/test_table_loader.py]: @jamie-cyber1 will fix this today

**Comment by birdage** [comment]
@jamie-cyber1 ready for merge

**Pull request merged**